library/mt_ip.py

```diff
@@ -44,9 +44,11 @@
     hostname:      "{{ inventory_hostname }}"
```

Comment:
line 30 for options of the parameter key do not match the code on line 67 at all